inc/ticket.class.php

```diff
@@ -5238,11 +5238,16 @@ static function showListForItem(CommonDBTM $item) {
             // you can only see your tickets
             if (!Session::haveRight(self::$rightname, self::READALL)) {
                $restrict .= " AND (`glpi_tickets`.`users_id_recipient` = '".Session::getLoginUserID()."'
-                                   OR (`glpi_tickets_users`.`tickets_id` = '".$item->getID()."'
+                                   OR (`glpi_tickets_users`.`tickets_id` = `glpi_tickets`.`id`
                                        AND `glpi_tickets_users`.`users_id`
                                             = '".Session::getLoginUserID()."')
                                    OR `glpi_groups_tickets`.`groups_id` IN (".implode(",",$_SESSION['glpigroups'])."))";
             }
+
+            if (Session::haveRight(self::$rightname, self::ASSIGN)) {
```

Comment:
No,

If you have the right 'READALL' : you see all the tickets : (all = assigned to me, to my groups, or others groups, and not assigned=new)

In own configuration, I have the rights :
 - "see my tickets"
 - assign (to assign new tickets)
(cf https://github.com/glpi-project/glpi/issues/1916)

So: 
- I don't see all tickets. 
- I see new tickets (for assign to me, to my group or other person) (I think because right "Ticket::ASSIGN:") : https://github.com/glpi-project/glpi/blob/9.1.2/inc/ticket.class.php#L272-L273
- I see my ticket

Thanks